ui/decredmaterial/icon_gallery.go

```diff
@@ -0,0 +1,189 @@
+	i.Mixer = NewImage(decredIcons["mixer"])
+	i.Next = NewImage(decredIcons["ic_next"])
+	i.SettingsIcon = NewImage(decredIcons["settings"])
+	i.SecurityIcon = NewImage(decredIcons["security"])
+	i.HelpIcon = NewImage(decredIcons["help_icon"])
+	i.AboutIcon = NewImage(decredIcons["about_icon"])
+	i.DebugIcon = NewImage(decredIcons["debug"])
+	i.VerifyMessageIcon = NewImage(decredIcons["verify_message"])
+	i.LocationPinIcon = NewImage(decredIcons["location_pin"])
+	i.AlertGray = NewImage(decredIcons["alert_gray"])
+	i.ArrowDownIcon = NewImage(decredIcons["arrow_down"])
+	i.WatchOnlyWalletIcon = NewImage(decredIcons["watch_only_wallet"])
+	i.CurrencySwapIcon = NewImage(decredIcons["swap"])
+	i.SyncingIcon = NewImage(decredIcons["syncing"])
+	i.DocumentationIcon = NewImage(decredIcons["documentation"])
+	i.Restore = NewImage(decredIcons["restore"])
+	i.TimerIcon = NewImage(decredIcons["timerIcon"])
+	i.WalletIcon = NewImage(decredIcons["wallet"])
+	i.StakeIcon = NewImage(decredIcons["stake"])
+	i.StakeIconInactive = NewImage(decredIcons["stake_inactive"])
+	i.StakeyIcon = NewImage(decredIcons["stakey"])
+	i.NewStakeIcon = NewImage(decredIcons["stake_purchased"])
+	i.TicketImmatureIcon = NewImage(decredIcons["ticket_immature"])
+	i.TicketUnminedIcon = NewImage(decredIcons["ticket_unmined"])
+	i.TicketLiveIcon = NewImage(decredIcons["ticket_live"])
+	i.TicketVotedIcon = NewImage(decredIcons["ticket_voted"])
+	i.TicketMissedIcon = NewImage(decredIcons["ticket_missed"])
+	i.TicketExpiredIcon = NewImage(decredIcons["ticket_expired"])
+	i.TicketRevokedIcon = NewImage(decredIcons["ticket_revoked"])
+	i.DecredSymbolIcon = NewImage(decredIcons["decred_symbol"])
+	i.DecredSymbol2 = NewImage(decredIcons["ic_decred02"])
+	i.GovernanceActiveIcon = NewImage(decredIcons["governance_active"])
+	i.GovernanceInactiveIcon = NewImage(decredIcons["governance_inactive"])
+	i.Rebroadcast = NewImage(decredIcons["rebroadcast"])
+
+	i.SettingsActiveIcon = NewImage(decredIcons["settings_active"])
+	i.SettingsInactiveIcon = NewImage(decredIcons["settings_inactive"])
+	i.ActivatedActiveIcon = NewImage(decredIcons["activated_active"])
+	i.ActivatedInactiveIcon = NewImage(decredIcons["activated_inactive"])
+	i.LockinActiveIcon = NewImage(decredIcons["lockin_active"])
+	i.LockinInactiveIcon = NewImage(decredIcons["lockin_inactive"])
+	i.TransactionFingerprint = NewImage(decredIcons["transaction_fingerprint"])
+	i.ArrowForward = NewImage(decredIcons["arrow_fwd"])
+
+	i.DexIcon = NewImage(decredIcons["dex_icon"])
+	i.DexIconInactive = NewImage(decredIcons["dex_icon_inactive"])
+	i.BTC = NewImage(decredIcons["dex_btc"])
+	i.DCR = NewImage(decredIcons["dex_dcr"])
+
+	return i
+}
+
+func (i *Icons) DarkModeIcons() *Icons {
+	decredIcons := assets.DecredIcons
+
+	i.OverviewIcon = NewImage(decredIcons["d_overview"])
+	i.OverviewIconInactive = NewImage(decredIcons["d_overview_inactive"])
+	i.WalletIconInactive = NewImage(decredIcons["d_wallet_inactive"])
```

Comment:
the naming convention for the dark mode icons seem off `overview_dark`, `overview_dark_inactive`, but considering the amount of work it would take to change this, we can leave as it, maybe during a future code cleanup, can be revisited

Reply:
i think `dm_overview` is more illustrative for a start.